### PR TITLE
(dropbox) Updated AU script and simplified installer script

### PR DESCRIPTION
**Pull request by The-Running-Dev**

### CODE REVIEW

File: automatic/dropbox/dropbox.nuspec [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+  <metadata>
+    <id>dropbox</id>
+    <version>22.4.24</version>
+    <title>Dropbox</title>
+    <authors>Dropbox</authors>
+    <owners>chocolatey, Rob Reynolds</owners>
+    <licenseUrl>http://www.dropbox.com/terms</licenseUrl>
+    <projectUrl>https://www.dropbox.com/</projectUrl>
+    <iconUrl>https://cdn.rawgit.com/ferventcoder/chocolatey-packages/02c21bebe5abb495a56747cbb9b4b5415c933fc0/icons/dropbox.svg</iconUrl>
```
Comment (AdmiringWorm) [inline automatic/dropbox/dropbox.nuspec]: the icon should be located in this repository

File: automatic/dropbox/dropbox.nuspec [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <description>Dropbox is a free service that lets you bring all your photos, docs, and videos anywhere. This means that any file you save to your Dropbox will automatically save to all your computers, phones and even the Dropbox website. Dropbox also makes it super easy to share with others, whether you're a student or professional, parent or grandparent. Even if you accidentally spill a latte on your laptop, have no fear! You can relax knowing that Dropbox always has you covered, and none of your stuff will ever be lost.
+
+### Notes
+
+Even though the official installer for Dropbox asserts administrator rights, it installs Dropbox only for the currently logged user (path: `%APPDATA%\Dropbox`).
+
+If the official Dropbox installer would be started with administrative rights, the installation would not be unattended.
+
+To prevent this problem, this package uses an AutoHotkey-script (thus the [autohotkey.portable](./autohotkey.portable) dependency) to close a non-unattended dialog box opened by the Dropbox installer during installation.
```
Comment (AdmiringWorm) [inline automatic/dropbox/dropbox.nuspec]: this is not true anymore, there is no autohotkey script present.
if the previous note about Dropbox installer can't be started with admin rights is no longer valid, it must be removed as well.

File: automatic/dropbox/dropbox.nuspec [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+The confusing dialog box with the content *“Dropbox will not work correctly if installed using the ‘Run as Administrator’ command. …”* can be ignored. Always select “Yes” if this dialog box appears. Regardless of this message, Dropbox will work correctly.</description>
+    <summary>Dropbox is a free service that lets you bring all your photos, docs, and videos anywhere.</summary>
+    <tags>dropbox virtual files file admin</tags>
```
Comment (AdmiringWorm) [inline automatic/dropbox/dropbox.nuspec]: you're missing the `freeware` and `cross-platform` tag.
See [1.2.2 Obligatory tags](https://github.com/chocolatey/chocolatey-coreteampackages/wiki/Contributing-guidelines#122-obligatory-tags)

File: automatic/dropbox/dropbox.nuspec [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+    <packageSourceUrl>https://github.com/ferventcoder/chocolatey-packages/</packageSourceUrl>
```
Comment (AdmiringWorm) [inline automatic/dropbox/dropbox.nuspec]: wrong package source url

File: automatic/dropbox/tools/chocolateyInstall.ps1 [hunk 5]
```diff
@@ -0,0 +1,17 @@
+﻿$packageArgs        = @{
+    packageName     = $env:ChocolateyPackageName
+    softwareName    = $evn:ChocolateyPackageTitle
```
Comment (AdmiringWorm) [inline automatic/dropbox/tools/chocolateyInstall.ps1]: please set the software name manually, don't rely on `ChocolateyPackageTitle` as this was first added in 0.10.1

File: automatic/dropbox/tools/chocolateyInstall.ps1 [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+    unzipLocation   = $env:ChocolateyPackageFolder
```
Comment (AdmiringWorm) [inline automatic/dropbox/tools/chocolateyInstall.ps1]: please remove `unzipLocation`, you have no need for it.

File: automatic/dropbox/update.ps1 [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+function global:au_GetLatest {
+  if ($global:getBetaVersion) {
+    
+    $betaVersion = ((Get-FirstBetaLink $global:betaVersionReleasePageUri $global:betaVersionRegEx) -replace $global:betaVersionRegEx, '$1') -replace '-', '.'
+    $betaVersionDownloadUri = $ExecutionContext.InvokeCommand.ExpandString($global:betaVersionDownloadUri)
+
+    return @{ URL32 = $betaVersionDownloadUri; Version = $betaVersion }
+  }
+
+  $stableVersionDownloadUri = ((Get-WebURL -Url $global:stableVersionDownloadUri).ResponseUri).AbsoluteUri
```
Comment (AdmiringWorm) [inline automatic/dropbox/update.ps1]: `Get-WebURL`?
Have you forgotten to create this function?

File: automatic/dropbox/dropbox.nuspec [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+  </metadata>
+</package>
```
Comment (AdmiringWorm) [inline automatic/dropbox/dropbox.nuspec]: you're missing the files element.

File: automatic/dropbox/update.tests.ps1 [hunk 9]
```diff
@@ -0,0 +1,120 @@
...
+        $versionInfo.URL32 | Should Match $installerFileNameRegEx
+    }
+
+    $global:getBetaVersion = $false
+  }
+
+  AfterAll {
+    $progressPreference = 'continue'
+  }
+}
```
Comment (AdmiringWorm) [inline automatic/dropbox/update.tests.ps1]: you have verified that this file isn't picked up by AU by a mistake?
you can test that by running `.\test_all.ps1 dropbox` in the root of this repository.

**Comment by The-Running-Dev** [comment]
I'm going to recreate the PR with the requested changes.

**Pull request closed without merging**